src/conf.c

```diff
@@ -847,6 +847,14 @@ config_backup(toml_table_t *conftab, backup_config_t *c, const char *instance,
 				status = false;
 			}
 
+		} else if (! strcasecmp("s3-connect-timeout", name)) {
```

Comment:
Only nit picky thing I can say it to keep the flags in the same order in both backup and restore

Reply:
They are but restore does not have s3_max_async_downloads so it looks different